source/appendices/a-glossary-of-terms.rst

```diff
@@ -729,7 +731,7 @@ Note: The main Code of Practice document takes precedence in the case of any con
      -
 
    * - Other_Free_to_Read
-     - A COUNTER Access_Type for institutional repositories. At the time of the transaction, the content item was freely available for reading (no license required) and did not qualify under the OA_Gold Access_Type.
+     - See Free_To_Read.
```

Comment:
Free_To_Read on longer contains a reference to Other_Free_To_Read, so maybe the change should be explained here?